backend/utils/resume_feedback.py

```diff
@@ -1,6 +1,6 @@
 import json
 from backend.schemas.user_input import UserInput
-from backend.utils.fit_score import calculate_fit_score
+from backend.schemas.categorical_keyword import CategoricalKeyword
```

Comment:
change this to module import 

from backend.schemas import Categorical Keyword